web/client/configs/localConfig.json

```diff
@@ -558,7 +558,7 @@
             "OmniBar", "Login", "Save", "SaveAs", "Expander", "Undo", "Redo", "FullScreen", "GlobeViewSwitcher", "SearchServicesConfig", "SearchByBookmark", "WidgetsBuilder", {
               "name": "Widgets",
               "cfg": {
-                "enableZoomInTblWidget": true
+                "enableZoomInTblWidget": false
```

Comment:
no need to add this in the cfg. the reason of that is because we want this functionality to be enabled by default (without changing the configuration) and if in certain project they want to disable it, they can because it's an optional config.

so please, avoid changing **localConfig.json** because it's not required  --> you just have to use a defaultProps for enableZoomInTblWidget inside first plugin component as you did, and that is enough for the mapstore framework (this repo/project)